avocado/utils/cpu.py

```diff
@@ -22,7 +22,7 @@
 
 import re
 import os
-
+import platform
```

Comment:
This is a minor (style only) issue, but still one that needs a fix.  Please add the `import platform` line, while keeping the two newlines after the import block.